letsencrypt-auto-source/tests/oraclelinux6_tests.sh

```diff
@@ -27,55 +24,31 @@ if [ $RESULT -ne 0 ]; then
   exit 1
 fi
 
-echo "PASSED: bootstrap from old letsencrypt-auto succeeded and installed Python 3.4"
+echo "PASSED: Bootstrap from old letsencrypt-auto succeeded and installed Python 3.4"
 
-# Expect letsencrypt-auto to just fail to rebootstrap and start certbot in interactive
-# shell since SCL is not installed.
-if ! "$LE_AUTO" --version 2>&1 | grep -q "Enable the SCL repository and try running Certbot again."; then
+# Expect letsencrypt-auto to skip rebootstrapping with a warning since SCL is not installed.
+if ! "$LE_AUTO" --non-interactive --version 2>&1 | grep -q "Skipping upgrade because new OS dependencies may need to be installed."; then
```

Comment:
nit: Can we check for (part of) `This requires manual user intervention: please run this script again manually.` here to test specifically for the interactive bootstrap case.

Reply:
Done.

Reply:
Done.